numpy/lib/tests/test_format.py

```diff
@@ -526,6 +527,7 @@ def test_load_padded_dtype(tmpdir, dt):
     assert_array_equal(arr, arr1)
 
 
+@pytest.mark.skipif(IS_WASM, reason="Emscripten NODEFS has a buggy dup")
```

Comment:
Xfail

Reply:
I've spent so much time fixing dup bugs in various Emscripten file systems. It's disappointing to keep seeing more of them.